framework/api/infrastructure/middleware.py

```diff
@@ -27,16 +28,42 @@ async def deserialise_web_request():
         _RequestData: dict = request.get_json()
 
         for _AttributeName, _AttributeType in get_type_hints(REQUEST_OBJECTS_BY_ENDPOINT[_RequestEndpoint]).items():
-            _Data = _RequestData[_AttributeName]
+            
+            _Data = _RequestData.get(_AttributeName)
```

Comment:
does .get() with just the key raise an exception if the key is not present?

Reply:
It returns None in that scenario. Also tested this scenario.

@overload 
 def get(self, __key: _KT) -> _VT | None: ...
 
*KT = KeyType
*VT = ValueType